tools/mcp/dotnet/AzureSDKMCPHub/Azure.SDK.Tools.MCP.Hub/Program.cs

```diff
@@ -1,37 +1,40 @@
+using Azure.SDK.Tools.MCP.Hub.Services.Azure;
 
-using System.Reflection;
-using Azure.SDK.Tools.MCP.Contract;
+namespace Azure.SDK.Tools.MCP.Hub;
 
-namespace Azure.SDK.Tools.MCP.Hub
+public sealed class Program
 {
-    public sealed class Program
+    public static void Main(string[] args)
     {
-        public static void Main(string[] args)
-        {
-            // todo: parse a command bundle here. pass it to CreateHostBuilder once we have an actual type
-            // todo: can we have a "start" verb and a <tool> verb? EG if someone calls <server.exe> HelloWorld
-            //   "This is a hello world input" -> we invoke just that tool
-            //   "<server.exe> start" -> runs server responding to vscode copilot chat 
-            CreateHostBuilder(args).Build().Run();
-        }
+        // todo: parse a command bundle here. pass it to CreateHostBuilder once we have an actual type
+        // todo: can we have a "start" verb and a <tool> verb? EG if someone calls <server.exe> HelloWorld
+        //   "This is a hello world input" -> we invoke just that tool
+        //   "<server.exe> start" -> runs server responding to vscode copilot chat
+        var host = CreateAppBuilder(args).Build();
+        host.MapMcp();
```

Comment:
All this refactoring is maybe not necessary. I just couldn't get the `MapMcp` wired up originally, but that was before I had pulled in the aspnetcore mcp package.